src/amo/components/MastHead.js

```diff
@@ -35,13 +35,13 @@ export class MastHeadBase extends React.Component {
         </div>
         {children}
         <header className="MastHead-header">
-          { isHomePage ?
-            <h1 ref={(ref) => { this.title = ref; }}
-                className="MastHead-title MastHead-homepage">
-              { headerTitle }
-            </h1> :
-            <a ref={(ref) => { this.title = ref; }}
-               href="/" className="MastHead-title">{ headerTitle }</a> }
+          {isHomePage
+            ? <h1 ref={(ref) => { this.title = ref; }} className="MastHead-title MastHead-homepage">
```

Comment:
I originally refactored into an `if` and `let` but it was enormous so I figured there must be a simpler way. This makes it more clear that there's a ternary going on I guess.